examples/c/wind_turbine_nmpc.c

```diff
@@ -523,6 +577,37 @@ int main()
 	external_function_param_casadi_create_array(NN, impl_ode_jac_x_xdot_u, np);
 	external_function_param_casadi_create_array(NN, impl_ode_fun_jac_x_xdot_u, np);
 	external_function_param_casadi_create_array(NN, impl_ode_jac_x_u, np);
+	// gnsf model
+	external_function_param_casadi_create_array(NN, phi_fun, np);
+	external_function_param_casadi_create_array(NN, phi_fun_jac_y, np);
+	external_function_param_casadi_create_array(NN, phi_jac_y_uhat, np);
+	external_function_param_casadi_create_array(NN, f_lo_jac_x1_x1dot_u_z, np);
+
+	int gnsf_num_stages = 5;
+	int gnsf_num_steps  = 2;
```

Comment:
Choosing the same opts as NEW_LIFTED_IRK (ns=4 and num_steps=1), and even requiring on average less SQP iterations than it, in this example GNSF is quite slower that NEW_LIFTED_IRK (11 vs 8 ms on my PC, Sandy-Bridge BLASFEO target). Any idea why?